readme.md

```diff
@@ -1,16 +1,16 @@
-![Renovate banner](https://app.renovatebot.com/images/renovate_660_220.jpg)
+![Renovate banner, paintbrush roller in a blue circle with the word Renovate next to it, green background](https://app.renovatebot.com/images/renovate_660_220.jpg)
```

Comment:
```suggestion
![Renovate banner](https://app.renovatebot.com/images/renovate_660_220.jpg)
```

Let's go back to the old Renovate banner text. For logos WebAIM recommends:

> Many websites link the main branding logo to the home page. Providing alternative text for the image, such as your company name (`alt="Acme Company"`), will usually suffice. The word "logo" (`alt="Acme Company Logo"`) is usually not an important part of the image's content or function. Similarly, it is usually not necessary to indicate that the image links to the home page (`alt="Acme Company home page"`) because this is a common convention—hearing "Link, Graphic, Acme Company" at the top of a web page is sufficient for a screen reader user to know that it is a logo linked to the home page.